WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -789,6 +822,12 @@ final class EditableOrderViewModel: ObservableObject {
         trackCustomerNoteAdded()
     }
 
+    /// Saves the current contents of the Order Note
+    ///
+    func saveInFlightOrderNotes() {
+        updateCustomerNote()
```

Comment:
We need some gate on this, we shouldn't sync the order notes (and track that they've changed) if there's no changes or we've previously synced them. Especially not repeatedly in response to rotation...

Reply:
True! I've updated it to use the same approach as with customer details, through checking the latest state of the note within the order synchronizer: https://github.com/woocommerce/woocommerce-ios/pull/12306/commits/a5b491a6038ef158e62c83908b9efbabb53340af